server/bundler/babel/babel-lodash-es/index.js

```diff
@@ -1,6 +1,6 @@
 /** @format */
 
-const types = require( 'babel-types' );
+const types = require( '@babel/types' );
```

Comment:
FYI, this could be deleted and moved to the line before as

    module.exports = function({ types }) {

Reply:
nice! I'll update that